lib/webpacker/dev_server_proxy.rb

```diff
@@ -0,0 +1,23 @@
+require "rack/proxy"
+
+class Webpacker::DevServerProxy < Rack::Proxy
+  def rewrite_response(response)
+    status, headers, body = response
+    headers.delete "transfer-encoding"
```

Comment:
Hi @gauravtiwari do you by any chance remember why you needed to delete the Transfer-Encoding header? Maybe something to do with preventing Gzipping?

Doing so breaks chunked streaming as per https://github.com/rails/webpacker/pull/2196#pullrequestreview-268686577

Currently my PR conditionally deletes the header under the assumption that in some cases it does need deleting, but as suggested it would be even better to get rid of the line if not needed :)